src/core/fonts.js

```diff
@@ -225,7 +225,8 @@ var ToUnicodeMap = (function ToUnicodeMapClosure() {
 
     forEach(callback) {
       for (var charCode in this._map) {
-        callback(charCode, this._map[charCode].charCodeAt(0));
+        if (typeof this._map[charCode].charCodeAt === 'function')
```

Comment:
> When seeing that error, no fonts are loaded... I thought dealing with the error directly would be better

I'm thinking dealing with the error directly will slow down font parsing and will make PDF.js slower.